swarm64_tpc_toolkit/correctness.py

```diff
@@ -7,6 +7,14 @@
 
 LOG = logging.getLogger()
 
+def RoundJSON(obj):
```

Comment:
Please:

- Move into `Correctness`
- Make it `@classmethod`
- Rename to `round_json`

Reply:
done